_maps/RandomRuins/LavaRuins/lavaland_surface_syndicate_base1.dmm

```diff
@@ -2890,7 +2890,7 @@
 	},
 /obj/effect/turf_decal/caution/red{
 	dir = 8;
-	pixel_x = 6
+	
```

Comment:
extraneous blank line, delete + remove `;` from end of previous line

Reply:
oops.